packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -62,7 +62,15 @@ if [[ -z $CONTAINER_REGISTRY ]]; then
   CONTAINER_REGISTRY=icr.io
 fi
 if [[ -z $DOCKER_IMAGE_NAME ]]; then
```

Comment:
Do we want to add some general docs or examples of how to use `DOCKER_IMAGE_NAME`? :) Maybe you have some local examples we can put into our docs.

Reply:
Good point, I added a comment for both DOCKER_IMAGE_NAME and LAYER_NAME.